src/gameengine.js

```diff
@@ -223,10 +228,38 @@ GameEngine.prototype = {
                 case 77: //M;
                     Howler.mute(that.muted = !that.muted);
                     break;
+                case 80: //P
+                    that.paused = !that.paused;
+                    break;
+                default:
+                    console.log(e.which);
             }
             //console.log(e.which);
             e.preventDefault();
         }, false);
+
+        this.ctx.canvas.addEventListener("blur", function() {
```

Comment:
This is just me and I get that it's convention but I've never liked blur. It's cleverness over readability (even if it's very slight) which just adds a little bit of unnecessary parsing to figure out what it is as opposed to "lostFocus" or "gainedFocus" or something. One term for one thing. Plus if we ever need the phrase "blur" for anything in the future, (like actually blurring the screen), this is verbal real estate that's been co-opted already.